maintainers/maintainer-list.nix

```diff
@@ -9214,6 +9214,12 @@
     githubId = 5124422;
     name = "Julien Urraca";
   };
+  justanotherariel = {
+    email = "ariel@ebersberger.io";
+    github = "justanotherariel";
+    githubId = 31776703;
+    name = "Ariel Ebersberger";
+  };
```

Comment:
I think I missed this the first time around, but this should go into it's own commit called: `maintainers: add justanotherariel`

Reply:
Ok, I will fix it as soon as we have decided on what to do with the download link